config/default.go

```diff
@@ -43,17 +43,18 @@ var defaultConfig = &Config{
 	Registry: Registry{
 		Backend: "consul",
 		Consul: Consul{
-			Addr:          "localhost:8500",
-			Scheme:        "http",
-			KVPath:        "/fabio/config",
-			TagPrefix:     "urlprefix-",
-			Register:      true,
-			ServiceAddr:   ":9998",
-			ServiceName:   "fabio",
-			ServiceStatus: []string{"passing"},
-			CheckInterval: time.Second,
-			CheckTimeout:  3 * time.Second,
-			CheckScheme:   "http",
+			Addr:               "localhost:8500",
+			Scheme:             "http",
+			KVPath:             "/fabio/config",
+			TagPrefix:          "urlprefix-",
+			Register:           true,
+			ServiceAddr:        ":9998",
+			ServiceName:        "fabio",
+			ServiceStatus:      []string{"passing"},
+			CheckInterval:      time.Second,
+			CheckTimeout:       3 * time.Second,
+			CheckScheme:        "http",
+			CheckTLSSkipVerify: false,
```

Comment:
Oh okay, thanks for the info! So just to clarify, should I remove this line entirely:

`f.BoolVar(&cfg.Registry.Consul.CheckTLSSkipVerify, "registry.consul.register.checkTLSSkipVerify", defaultConfig.Registry.Consul.CheckTLSSkipVerify, "service check TLS verifcation")`

or set it to:

`f.BoolVar(&cfg.Registry.Consul.CheckTLSSkipVerify, "registry.consul.register.checkTLSSkipVerify", false, "service check TLS verifcation")`

or remove it completely?

And I'll update fabio.properties in my next push. Sorry about that.